src/ui/public/modals/confirm_modal.html

```diff
@@ -1,32 +1,10 @@
-<div class="kuiModal" style="width: 450px" data-test-subj="confirmModal">
-  <div class="kuiModalHeader" ng-if="title">
-    <div class="kuiModalHeader__title" data-test-subj="confirmModalTitleText">
-      {{title}}
-    </div>
-  </div>
-  <div class="kuiModalBody">
-    <div
-      class="kuiModalBodyText"
-      data-test-subj="confirmModalBodyText"
-    >
-      {{message}}
-    </div>
-  </div>
-
-  <div class="kuiModalFooter">
-    <button
-      class="kuiButton kuiButton--hollow"
-      data-test-subj="confirmModalCancelButton"
-      ng-click="onCancel()"
-    >
-      {{cancelButtonText}}
-    </button>
-    <button
-      class="kuiButton kuiButton--primary"
-      data-test-subj="confirmModalConfirmButton"
-      ng-click="onConfirm()"
-    >
-      {{confirmButtonText}}
-    </button>
-  </div>
-</div>
+<confirm-modal
```

Comment:
In that case, I think we should stick with this path, so we ensure modules from the uiFramework are only morphed into directives once.

In either case, if we want to have a longer discussion about this, I say we move it out of this specific PR since it's not something new that this PR introduced.